src/rviz_visual_tools.cpp

```diff
@@ -1077,6 +1077,58 @@ bool RvizVisualTools::publishPolygon(const geometry_msgs::Polygon &polygon, cons
   publishPath(points, color, scale, ns);
 }
 
+  bool RvizVisualTools::publishWireframeCuboid(const Eigen::Affine3d &pose,
+					       const Eigen::Vector3d &position,
+					       const Eigen::Matrix3d &rotation_matrix,
+					       const Eigen::Vector3d &min_point,
+					       const Eigen::Vector3d &max_point,
+					       const RvizVisualTools::colors &color=RvizVisualTools::BLUE)
```

Comment:
these should be documented in the header file - why do i need both pose and position and rotation matrix?